# Light Animations

int8 LEFT_TURN = 1
int8 RIGHT_TURN = 2
int8 REVERSE = 3
int8 BLOCKED = 4
int8 UP = 5
int8 DOWN = 6

# Selects which animation to execute

int8 light_animation